src/databricks/labs/dqx/rule.py

```diff
@@ -34,25 +35,32 @@ class ExtraParams:
     """Class to represent extra parameters for DQEngine."""
 
     column_names: dict[str, str] = field(default_factory=dict)
+    run_time: datetime = datetime.now()
 
 
 @dataclass(frozen=True)
 class DQRule:
     """Class to represent a data quality rule consisting of following fields:
-    * `check` - Column expression to evaluate. This expression should return string value if it's evaluated to true -
-    it will be used as an error/warning message, or `null` if it's evaluated to `false`
+    * `check_func` - check function to be applied
+    * `col_name` - column name to which the given check function should be applied
     * `name` - optional name that will be given to a resulting column. Autogenerated if not provided
```

Comment:
Docstring needs to be updated. Sth like "optional name that will be given to the resulting check name. "